plat/xen/arm/setup64.c

```diff
@@ -300,6 +300,13 @@ void _libxenplat_armentry(void *dtb_pointer, paddr_t physical_offset)
 	if (unlikely(r))
 		UK_CRASH("Failed to initialize bootstrapping CPU: %d\n", r);
 
+#ifdef CONFIG_HAVE_SMP
+	ret = lcpu_mp_init(CONFIG_UKPLAT_LCPU_RUN_IRQ,
```

Comment:
I think the `-` in the commit message should be replaced by a comma

Reply:
fixed